examples/progress-dashboard/app.py

```diff
@@ -0,0 +1,205 @@
+# app dashboard from https://huggingface.co/spaces/davanstrien/argilla-progress/blob/main/app.py
+import os
+from typing import List
+
+import argilla as rg
+import gradio as gr
+import pandas as pd
+import plotly.colors as colors
+import plotly.graph_objects as go
+
+client = rg.Argilla(
+    api_url=os.getenv("ARGILLA_API_URL"),
+    api_key=os.getenv("ARGILLA_API_KEY"),
+)
+
+
+def get_progress(dataset: rg.Dataset) -> dict:
+    dataset_progress = dataset.progress(with_users_distribution=True)
+
+    total, completed = dataset_progress["total"], dataset_progress["completed"]
+    progress = (completed / total) * 100 if total > 0 else 0
+
+    return {
+        "total": total,
+        "annotated": completed,
+        "progress": progress,
+        "users": {
+            username: user_progress["completed"].get("submitted")
+            for username, user_progress in dataset_progress["users"].items()
+        },
+    }
+
+
+def create_gauge_chart(progress):
```

Comment:
should we add a legend to this chart? It isn't too clear what the colors red vs blue mean.